tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Execute Batch Payload operations using Java Database Connectivity (JDBC) Receiver Adapter
+description: Learn how to connect, interact, and perform batch operations on your database via JDBC receiver adapter.
+keywords: tutorial
+auto_validation: true
+time: 40
+tags: [ tutorial>intermediate, software-product>sap-integration-suite]
+primary_tag: software-product>sap-business-technology-platform
+parser: v2
+---
+
+## You will learn
+-	To connect Java Database Connectivity (JDBC) receiver adapter with your database (on the receiver system) and send, receive, or update data on the database.
+-	To interact with your database using SQL queries and XML.
+-	To perform operations on database using Batch Payload functionality.
+
+
+## Prerequisites
+
+ - You have enabled the Cloud Integration capability of [SAP Integration Suite](http://bit.ly/3VmWOOZ).
+ - You have configured your [JDBC adapter](https://blogs.sap.com/2018/10/31/configuring-jdbc-adapter-a-step-by-step-guide/).
+
+
+## Intro
+The data that you are transferring (sending or receiving) is called payload. The payload sent to the JDBC adapter can be modified. You can modify multiple records in a single payload using either INSERT, UPDATE, or DELETE modes. This can be achieved using Batch Payload functionality.
+
+There are two types of Batch operations:
+
+-	Atomic: This mode considers each batch operation as a single unit. It updates the whole batch operation successfully or reverts the entire operation to its initial state if anything in the batch operation fails.
+
+-	Non-Atomic- The operation in this mode varies depending upon the database and its driver. Generally, it updates all the successfully executed records and throws an exception if anything fails. It does not revert the failed records to its initial state.
+
+Following the steps below, you will be able to setup a sample database, create an integration flow, upload drivers, and data sources required for JDBC receiver adapter. Then, you can create a payload, execute, and benefit from Batch Payload functionality.
+
+
+### Choose your Database
```

Comment:
please use sentence case

Reply:
The words, JDBC, Batch Payload have to be in title case. These are product names/functionalities.